java/test/jmri/jmrix/cmri/serial/cmrimetrics/PackageTest.java

```diff
@@ -0,0 +1,36 @@
+package jmri.jmrix.cmri.serial.cmrimetrics;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+    CMRInetMetricsCollectorTest.class,
+    CMRInetMetricsDataTest.class
+})
+
+/**
+ * Tests for the jmri.jmrix.cmri.serial.cmrimetrics package
+ *
+ * @author  Chuck Catania	Copyright (C) 2017
+ */
+public class PackageTest{
+
+    // Main entry point
```

Comment:
Please remove this main method. ant and  runtest.csh can run this package test (any test class even) without requiring this main method (this suppresses even normal success output from ant).

Reply:
Do I completely remove the file?  Other test files in the cmri test class have this method.  I followed a common model for making the tests.

Reply:
The test class was removed.  The main class is not implemented (yet) in this revision.

Reply:
The JUnit 4-based test suites (like this one) need no body to the class, so you want everything before the class declaration and the class to just be:

```
public class PackageTest {}
```

Reply:
Definitely don't remove the packagetest file.

You MUST have at least one test in the package for this to pass CI tests.

Reply:
Sorry that was probably not clear enough. You want something like this (plus any comments):

```
package jmri.jmrix.cmri.serial.cmrimetrics;

import org.junit.runner.RunWith;
import org.junit.runners.Suite;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

@RunWith(Suite.class)
@Suite.SuiteClasses({
    CMRInetMetricsCollectorTest.class,
    CMRInetMetricsDataTest.class
})

/**
 * Tests for the jmri.jmrix.cmri.serial.cmrimetrics package
 *
 * @author Chuck Catania Copyright (C) 2017
 */
public class PackageTest {}
```